install/update_92_93.php

```diff
@@ -728,6 +728,10 @@ function update92to93() {
    /** Add Item_Device* display preferences */
    $itemDeviceTypes = Item_Devices::getDeviceTypes();
    foreach ($itemDeviceTypes as $itemDeviceType) {
+
+      if (!class_exists($itemDeviceType)) {
+         continue;
+      }
```

Comment:
I guess `Item_Devices::getDeviceTypes()` should not return types that have no corresponding class.
Why is it necessary ?

Reply:
No, as far as I've seen, class existance is not checked before

Reply:
Indeed, but why this check is necessary ?
If a device return a bad value on its `getItem_DeviceType` method (which is the method used to populate `Item_Devices::getDeviceTypes()` result, then maybe it should be fixed there.

Reply:
I've removed; all seems to be ok

Reply:
:+1: